server/data/quotes.json

```diff
@@ -1,4 +1,8 @@
 ﻿[
+     {
+    "sentence": "As a hawk cannot fly high with one wing,similarly a man cannot reach excellence with one tongue.",
```

Comment:
Minor detail, but add a space in between the comma. :)

Reply:
ok